ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/attestation/MatchingDataAttestationGroup.java

```diff
@@ -56,14 +55,14 @@ public class MatchingDataAttestationGroup implements Iterable<ValidatableAttesta
   private final Spec spec;
   private Optional<Bytes32> committeeShufflingSeed = Optional.empty();
   private final AttestationData attestationData;
-  private final Supplier<Int2IntMap> commiteesSizeSupplier;
+  private final Optional<Int2IntMap> committeesSize;
```

Comment:
i think maybeCommitteeSize would be cleaner

Reply:
Left it as it is because we use that name in other places (where it is still `Optional`)